docs/eventing/experimental-features.md

```diff
@@ -51,4 +51,33 @@ data:
 
 ## Features list
 
-<!-- TODO there are no experimental features at the moment -->
+### `KReference.Group` field
+
+**Flag name**: `kreference-group`
+
+**State**: Alpha, disabled by default
+
+**Tracking issue**: https://github.com/knative/eventing/issues/5086
```

Comment:
If the feature is implemented as it's documented for now I don't think there's a need to add this. If it's not implemented, the docs shouldn't be added until it is.

Reply:
But this is an experimental feature, that is a feature which development is ongoing. Hence, to me it seems reasonable to link where the development is tracked. As an example, the Rust community does the same: https://doc.rust-lang.org/std/alloc/struct.Layout.html#method.for_value_raw

Reply:
OK